docs/pages/reference/terraform-provider/resources/installer.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Reference for the teleport_installer Terraform resource
+description: This page describes the supported values of the teleport_installer resource of the Teleport Terraform provider.
+---
+
+{/*Auto-generated file. Do not edit.*/}
+{/*To regenerate, navigate to integrations/terraform and run `make docs`.*/}
+
+
+
+## Example Usage
+
+```hcl
+# Teleport Installer resource
+
+resource "teleport_installer" "example" {
+  version = "v1"
+  metadata = {
+    name        = "example"
+    description = "Example Teleport Installer"
+
+    expires = "2024-10-12T07:20:50Z"
+
+    labels = {
+      example = "yes"
+    }
+  }
+
+  spec = {
+    # This is the default installer script. For more details about the installer script
+    # see https://goteleport.com/docs/enroll-resources/auto-discovery/servers/ec2-discovery/#step-67-optional-customize-the-default-installer-script
+    script = <<EOF
+#!/usr/bin/env sh
+set -eu
+
+cdnBaseURL='https://cdn.teleport.dev'
+teleportVersion='v16.1.8'
```

Comment:
This will quickly become outdated, can we use the version variable here? They're defined somewhere in config.js and are used throughout the documentation.